ReText/config.py

```diff
@@ -229,6 +253,8 @@ def saveSettings(self):
 				value = configurator.currentData()
 			elif isinstance(configurator, FileSelectButton):
 				value = configurator.fileName
+			elif isinstance(configurator, DirectorySelectButton):
+				value = configurator.fileName
```

Comment:
If you make a base class like I suggested above, then it will be possible to list only that class here, instead of two classes.